package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ddg-screen-diff",
```

Comment:
I'm going to scope this under our org, e.g. `@duckduckgo/ddg-screen-diff`.

I'll do that after this PR gets merged, this will need to be released to NPM anyway.

Reply:
Perfect 👌